HRIS.Services/Interfaces/IEmployeeTypeService.cs

```diff
@@ -9,19 +9,19 @@ public interface IEmployeeTypeService
     /// </summary>
     /// <param name="employeeTypeDto"></param>
     /// <returns></returns>
-    Task<EmployeeTypeDto> SaveEmployeeType(EmployeeTypeDto employeeTypeDto);
+    Task<EmployeeTypeDto> CreateEmployeeType(EmployeeTypeDto employeeTypeDto);
 
     /// <summary>
     ///     Delete Employee Type
     /// </summary>
     /// <param name="name"></param>
     /// <returns></returns>
```

Comment:
```suggestion
        /// <summary>
        ///     Deletes an existing employee type by id.
        /// </summary>
        /// <param name="id">The id of the employee type to delete.</param>
        /// <returns>The deleted EmployeeTypeDto object.</returns>
```

Just requires some updates on summary info for all